src/NetworkConfiguration.py

```diff
@@ -54,6 +54,6 @@ def parse_constants(constants):
     network_config_map['BLOCK_TIME_IN_SEC'] = int(constants['time_between_blocks'][0])
     network_config_map['BLOCKS_PER_CYCLE'] = int(constants['blocks_per_cycle'])
     network_config_map['BLOCKS_PER_ROLL_SNAPSHOT'] = int(constants['blocks_per_roll_snapshot'])
-    network_config_map['BLOCK_REWARD'] = int(constants['baking_reward_per_endorsement'][0] * constants['endorsers_per_block'])
+    network_config_map['BLOCK_REWARD'] = int(int(constants['baking_reward_per_endorsement'][0]) * int(constants['endorsers_per_block']))
```

Comment:
Is the wrapper int really necessary?

Reply:
Not a blocker.

Reply:
The outer one? Probably not, but fits the pattern of all the other parameters so i left it.